### PR TITLE
use .env/env-vars to set multinet api host

### LINKED ISSUE #76: Use an environment variable to set the Multinet host at build time

### PR DESCRIPTION
**Pull request by JackWilb**
Depends on #85

Closes #76 

You can test each file with `yarn serve` and `yarn serve --mode=production`

`yarn serve` defaults to development mode
`yarn build` defaults to production mode

[Reference](https://cli.vuejs.org/guide/mode-and-env.html#modes)

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+VUE_APP_MULTINET_API_URL=https://api.multinet.app/api
```
Comment (waxlamp) [inline .env:1]: For consistency with the client, this should be `VUE_APP_MULTINET_HOST`.

**Comment by JackWilb** [comment]
I think this would work without the netlify.toml, because the .env.production gets read automatically into the environment when the mode is production, which it is when you run `yarn build`. Not sure if you'd like to delete it, for now. We could also remove the .env.production in favor of the netlify.toml

**Comment by waxlamp** [comment]
> I think this would work without the netlify.toml, because the .env.production gets read automatically into the environment when the mode is production, which it is when you run `yarn build`. Not sure if you'd like to delete it, for now. We could also remove the .env.production in favor of the netlify.toml

Is this something `yarn` does automatically? The [Netlify docs on environment variables](https://docs.netlify.com/configure-builds/environment-variables/#declare-variables) don't mention `.env.*` files as a way to set these up. I think it would be better to use `netlify.toml` to control this, since it also allows you to tune variables and other settings based not just on build-vs-dev but also on deploy context.

So I would say let's get rid of `.env.production` and let `netlify.toml` be the mechanism that controls how Netlify builds things.

**Comment by JackWilb** [comment]
> Is this something `yarn` does automatically? T

Yes it is.

> So I would say let's get rid of `.env.production` and let `netlify.toml` be the mechanism that controls how Netlify builds things.

Agreed, see 886a351

File: README.md [hunk 2]
```diff
@@ -10,8 +10,9 @@ To get the application up and running in dev mode:
 
 1. Ensure that you have Node and yarn installed.
 2. Check out this repository and move into it (e.g., `cd view-adjmatrix`).
-3. Install the Yarn dependencies: `yarn install`.
-4. Run the application: `yarn serve`.
+3. Copy the correct .env.development to .env with `cp .env.development .env`
```
Comment (waxlamp) [inline README.md:13]:
```suggestion
3. Copy .env.development to .env with `cp .env.development .env`
```

**Review by waxlamp — DISMISSED** [review]
One suggested change; otherwise looks good.

**Pull request merged**